R/geo.R

```diff
@@ -105,8 +119,25 @@ addGeoMetadata <- function (variable) {
              " wrong, or Crunch doesn't yet have geodata for this variable.")
     }
     if (nrow(match_scores) > 1) {
-        halt("There is more than one possible match. Please specify the geography manually:\n",
-             paste0(capture.output(print(match_scores[,c("value", "geodatum_name", "geodatum", "property")])), collapse = "\n"))
+        # if there is a single geodatum where multiple properties match equally 
+        # well then grab the first property and use that since they are likely 
+        # duplications of the same feature.
+        if (length(unique(match_scores$geodatum_name)) == 1 &
+            length(unique(match_scores$geodatum)) == 1) {
+            warning("The geodatum ", dQuote(unique(match_scores$geodatum_name)),
+                    " has multiple properties that match the variable (",
+                    serialPaste(dQuote(match_scores$property)),
+                    "). Using ", dQuote(match_scores$property[1]), ".")
+            match_scores <- match_scores[1,]
+        } else {
+            halt("There is more than one possible match. Please specify the ", 
```

Comment:
PR seems generally ok to me, though it's too bad that there's no test coverage for this "else" case. Aside from the stats (which could be massaged with different line breaking in the code), my concerns are (1) that the error message construction itself could fail, and (2) that even with the error message I wouldn't know how to "specify the geography manually", and it's not clear to me how the output of that message would help me to know how. 

That said, I recognize that this error message itself isn't new in this PR. And in terms of how much it's worth spending to improve it: how likely is it that someone could get here?